README.md

```diff
@@ -101,8 +102,23 @@ The released version of the SDK can be downloaded from https://github.com/Appdyn
     * [C++ SDK and Sample App](https://github.com/Appdynamics/iot-cpp-sdk)
     * [Python Sample Code](https://github.com/Appdynamics/iot-rest-api-sample-apps)
 
+## Miscellaneous
+* Code Coverage 
+    * `jacoco` is integrated in the `build.gradle` file to track code coverage
```

Comment:
Delete this line. It's not really a bullet item as much as information about how Gradle generates code coverage.